iib/workers/tasks/opm_operations.py

```diff
@@ -89,7 +89,7 @@ def opm_serve(catalog_dir: str) -> Tuple[int, subprocess.Popen]:
 
     for port in _gen_port_for_grpc():
         try:
-            cmd = ['opm', 'serve', catalog_dir, '-p', str(port), '-t', '/dev/null']
+            cmd = [opm_path, 'serve', catalog_dir, '-p', str(port), '-t', '/dev/null']
```

Comment:
I am missing `global opm_path` in these functions (here and below)

Reply:
Even without the` global opm_path` I am able to use the opm_path set in set_opm_verison()

Reply:
Note: use global keyword is necessary inside a function when you want to modify the variable. For accessing it is not necessary.
----------------------------
I leave it up to you.